.pre-commit-config.yaml

```diff
@@ -13,6 +13,15 @@ repos:
     rev: 4.0.1
     hooks:
       - id: flake8
+  - repo: https://github.com/PyCQA/isort
+    rev: 5.10.1
+    hooks:
+      - id: isort
+  - repo: https://github.com/asottile/pyupgrade
+    rev: v2.34.0
+    hooks:
+      - id: pyupgrade
+        args: [--py36-plus]
```

Comment:
#26 covers this.

Reply:
AIUI (Andrew can correct me if this is wrong) we don't run 3.6 on our infra but the firefox build system imports taskgraph and we don't want to break that without some notice/coordination.

Reply:
Yeah, that is correct.

We could probably refactor the Gecko build system to avoid importing Taskgraph at the top-level.. but tbh, supporting 3.6 awhile longer seems easiest to me.